operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/KeycloakSpec.java

```diff
@@ -56,13 +57,14 @@ public class KeycloakSpec {
             "The special value `" + Constants.INSECURE_DISABLE + "` disables https.")
     private String tlsSecret;
 
-    @JsonPropertyDescription("Disable the default ingress.")
-    private boolean disableDefaultIngress;
-
     @JsonPropertyDescription(
             "In this section you can configure podTemplate advanced features, not production-ready, and not supported settings.\n" +
                     "Use at your own risk and open an issue with your use-case if you don't find an alternative way.")
     private KeycloakSpecUnsupported unsupported;
+    @JsonProperty("ingress")
+    @JsonPropertyDescription("The deployment is, by default, exposed through a basic nginx ingress.\n" +
+            "You can change this behaviour by setting the enabled property to false.")
```

Comment:
Based on suggestion from @pedroigor.
```suggestion
    @JsonPropertyDescription("The deployment is, by default, exposed through a basic ingress.\n" +
            "You can change this behaviour by setting the enabled property to false.")
```

Reply:
@vmuzikar changed accordingly. Thanks!